tests/test_lanes.py

```diff
@@ -528,6 +531,7 @@ def test_works_query(self):
         # Works in the series are returned as expected.
         w1 = self._work(with_license_pool=True)
         w1.presentation_edition.series = series_name
+        self._db.commit()
```

Comment:
We need to commit to make sure the series name shows up in the materialized view. Previously this wasn't an issue because we didn't use the materialized view to filter the series name; we did an aliased join against Edition and filtered that.